var/spack/repos/builtin/packages/py-azure-core/package.py

```diff
@@ -15,6 +15,7 @@ class PyAzureCore(PythonPackage):
 
     license("MIT")
 
+    version("1.30.2", sha256="a14dc210efcd608821aa472d9fb8e8d035d29b68993819147bc290a8ac224472")
```

Comment:
It's a gray area. Python 3.7 has been EOL for over a year (since 2023-06-27) and all 3.7 versions in spack have been deprecated since then too, but they are still available if you explicitly install deprecated versions. In practice, it depends on the contributor and maintainer whether the requirement for 3.8 is explicitly added for version upgrades.

Reply:
I'll just add them. At worst, it gets removed quickly when the python 3.7 versions are removed.